proto/engine/v1/BUILD.bazel

```diff
@@ -50,7 +61,7 @@ ssz_gen_marshal(
 go_proto_library(
     name = "go_proto",
     compilers = [
-        "@com_github_prysmaticlabs_protoc_gen_go_cast//:go_cast_grpc",
+"@com_github_prysmaticlabs_protoc_gen_go_cast//:go_cast_grpc",
```

Comment:
Can you revert this line? 

I know gazelle can't run on this file, so it's hard to maintain the formatting.